src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
It really does not work. See for yourself:
branch: https://github.com/kovalikp/nuproj/commits/test/common_targets
changes: https://github.com/kovalikp/nuproj/commit/d44060eaa6f405e3784dd251a1e03d895f2a5369
build: https://ci.appveyor.com/project/kovalikp/nuproj/build/1.0.67/tests